package.json

```diff
@@ -11,6 +11,7 @@
   "main": "umd/skatejs.js",
   "module": "es/index.js",
   "es2015": "es-latest/index.js",
+  "esnext": "src/index.js",
```

Comment:
Yeah, that works too. After I did this work, I thought that it might actually make more sense to point `esnext` at a version of the source that's transpiled to the latest JS spec (there are a few things that are used in Skate that aren't actually part of JS yet, like object spread).

If you want to remove `es2015` that's fine with me, but technically the semantics between `es2015` and `esnext` are _slightly_ different, as the blog post mentions.